doc/classes/CanvasItem.xml

```diff
@@ -411,6 +411,18 @@
 				Returns the canvas item RID used by [RenderingServer] for this item.
 			</description>
 		</method>
+		<method name="get_canvas_layer" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the layer index of the [CanvasLayer] that this node is in. Returns [code]0[/code] if this node is not in any [CanvasLayer].
+			</description>
+		</method>
+		<method name="get_canvas_layer_node" qualifiers="const">
+			<return type="CanvasLayer" />
+			<description>
+				Returns the [CanvasLayer] that this node is in, or [code]null[/code] if the node is not in any [CanvasLayer].
```

Comment:
I previously commented on this https://github.com/godotengine/godot/pull/87095#discussion_r1450273251 but it went unseen, probably.

Reply:
Sorry it might went unseen while i was updating the code. I'll change this into "Returns the [CanvasLayer] that contains this node."

Thanks!